terraform/stacks/tooling/cross_account_iam.tf

```diff
@@ -0,0 +1,54 @@
+resource "aws_iam_role" "payer_account_access" {
+    name = "payer-account-access"
+
+    assume_role_policy = <<EOF
+    {
+      "Version": "2012-10-17",
+      "Statement": [
+        {
+          "Effect": "Allow",
+          "Principal": {
+            "AWS": "arn:aws:iam::${var.payer_account_id}:root"
```

Comment:
Probably same issue here :)

Reply:
I don't think so here - here we know they are a commercial account, since they're the payer :)

Reply:
Ok, sounds good then

Reply:
Looks like you're trying to delegate access between an account in the `aws` partition and the `aws-us-gov` partition, which you cannot do :(.   https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_identifiers.html : "You cannot delegate access between accounts in different partitions."